tests/e2e/default-mobile/old-navigation/old-navigation.wdio-spec.js

```diff
@@ -52,58 +50,52 @@ describe('Old Navigation', () => {
   });
 
   it('should navigate to the Messages section and open a sent message', async () => {
-    await browser.pause(100);
-    /* const message = 'Navigations test';
+    const message = 'Navigations test';
     await oldNavigationPage.goToMessages();
```

Comment:
Try removing this like ([Slack conversation](https://medic.slack.com/archives/C02GME3BQE6/p1724379244272189?thread_ts=1724365554.039299&cid=C02GME3BQE6))

Reply:
Done